src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1485,6 +1497,21 @@ void ThrowIfNotProvidedForInstall(string attributeValue, string attributeName)
                 }
             }
 
+            void ThrowIfTagsFormatIsUnacceptable(string attributeValue, string attributeName)
```

Comment:
A bit more concise, no need to enter the DelimitedTextToDictionary if the value is null or whitespace:

```
void ThrowIfTagsFormatIsUnacceptable(string attributeValue, string attributeName)
{
    if(string.IsNullOrWhiteSpace(attributeValue))
    {
        return;
    }
   
    try
    {
        DelimitedTextToDictionary(attributeValue, "=", ",");
    }
    catch
    {
        throw new ValidationException($"{attributeName} is specified in incorrect format. Try as TagName=TagValue,TagName=TagValue in double quotes", false);
    }
}
```